app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/btle/profiles/battery/BatteryInfo.java

```diff
@@ -19,9 +19,10 @@
 import android.os.Parcel;
 import android.os.Parcelable;
 
-public class BatteryInfo implements Parcelable{
+public class BatteryInfo implements Parcelable {
 
     private int percentCharged;
+    private String address;
```

Comment:
I think this is not part of the spec, so it should be either avoided or marked at least.